### DISCUSSION TITLE
.bashrc and .bash_profile inconsistency.

### DISCUSSION THREAD

**Discussion by snuggs** [termux/termux-app#3731] [Q&A]
Wondering why Termux explicitly loads `~/.bashrc` (ONLY) from `$PREFIX/etc/bash.bashrc`.
However...`~/.bash_profile` ALSO gets loaded AFTERWARDS.

At minimum would like to know the call site of the latter. 

At maximum wondering why TERMUX chooses to load `~/.bashrc` when the standard LINUX convention is for `*/.(bash_)profile` to be loaded BY OS and to add explicit loads of `.bashrc` to be included from the profile.

This makes portability of loading `.bashrc` a potential infinite loop.

Any help would be appreciated. At least would like to know where / what is loading my `.bash_profile`.

Thanks in advance!

👇🏾😈

![Screenshot_20231211-071955_Termux](https://github.com/termux/termux-app/assets/38223/31ca9f0a-cba1-42c0-bc2b-e043b49a2b5e)

**Comment by agnostic-apollo** [comment]
Related https://github.com/termux/termux-packages/pull/7981
